src/StudyEntities/Candidate/CandID.php

```diff
@@ -0,0 +1,64 @@
+<?php declare(strict_types=1);
+/**
+ * This defines what a CandID should be so it can be passed as a typed
+ * parameter in functions.
+ *
+ * PHP Version 7
+ *
+ * @category StudyEntities
+ * @package  LORIS
+ * @author   Xavier Lecours Boucher <xavier.lecoursboucher@mcgill.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+namespace LORIS\StudyEntities\Candidate;
+
+/**
+ * A representation of a CandID object which is always a string of length 6
```

Comment:
```suggestion
 * A representation of a CandID object. A CandID is always a string of length 6.
```

I think this wording is slightly clearer.